libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -95,6 +96,8 @@ for FrSky SPort Passthrough
 #define AP_ARMED_OFFSET             8
 #define AP_BATT_FS_OFFSET           9
 #define AP_EKF_FS_OFFSET            10
+#define AP_IMU_TEMP_LIMIT           0x3F
+#define AP_IMU_TEMP_OFFSET          25
```

Comment:
Shouldn't AP_IMU_TEMP_OFFSET have a value of 26, if the corresponding value is populated in the 6 MSBs of 0x5001?
For example, compare to HOME_BEARING_OFFSET which has a value of 25, and the corresponding value is coded on the last 7 MSBs of 0x5004.

Reply:
As discussed above, I would add this definition:
#define AP_IMU_TEMP_ORIGIN 19